src/pages/workspaces/workspace/jobHistory/WorkflowDashboard.js

```diff
@@ -102,7 +103,7 @@ const WorkflowDashboard = _.flow(
   /*
    * Page render
    */
-  const { calls, end, failures, start, status, workflowLog, workflowName, submittedFiles: { workflow: wdl } = {} } = workflow || {}
+  const { metadataArchiveStatus, calls, end, failures, start, status, workflowLog, workflowName, submittedFiles: { workflow: wdl } = {} } = workflow || {}
```

Comment:
Will this throw an error if field `metadataArchiveStatus` is not present in the metadata response?  Or the value for metadataArchiveStatus is set to `undefined`?

Reply:
If it's not there it will be `undefined`